src/Types/Todo.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from './User';
+
+export interface Todo {
```

Comment:
You can extend ServerTodo interface just to add single user

Reply:
@sintax1s yep, let's extend `ServerTodo`